src/trans.ts

```diff
@@ -1,65 +1,67 @@
-import { get, reduce, reject } from 'lodash';
-import { OPTIONS } from './i18n-json-webpack-loader';
+import { get, reduce } from 'lodash';
 
-const isCreateElement = (entity) => get(entity, 'callee.property.name') === 'createElement';
+export const isElement = entity =>
+  get(entity, 'type') === 'CallExpression' &&
+  get(entity, 'callee.property.name') === 'createElement';
 
-const checkTransOn = (obj) => (
-  get(obj, 'type') === 'MemberExpression'
-  && get(obj, 'property.name') === 'Trans'
-);
+export const isTrans = entity =>
+  isElement(entity) &&
+  (get(entity, 'arguments[0].name') === 'Trans' ||
+    get(entity, 'arguments[0].property.name') === 'Trans');
 
-const isTrans = (args) => args
-  && checkTransOn(args[0])
-  || checkTransOn(args);
+const isInterpolation = entity =>
+  get(entity, 'type') === 'ObjectExpression' &&
+  get(entity, 'properties').length === 1 &&
```

Comment:
is this true... does it always only have 1 property?

Reply:
My reasoning was as follows: A valid Trans interpolation is specified as `{{variableName}}`, which in JSX expands to `{{ variableName: variableName }}`. Specifying multiple properties on the object is not valid interpolation syntax.

At some point a call to `createElement` is going to receive that object as one of its `child` arguments, at which point we'll add it to the term.